Modules/Sources/WordPressReader/Comments/Views/WebCommentContentRenderer.swift

```diff
@@ -36,26 +36,10 @@ public final class WebCommentContentRenderer: NSObject, CommentContentRenderer {
     private var lastReloadDate: Date?
     private var isReloadNeeded = false
 
-    /// A shared web view context with resources that can be reused across
-    /// mutliple web view instances.
-    @MainActor
-    public final class Context {
-        let processPool = WKProcessPool()
-
-        public init() {}
-    }
-
     // MARK: Methods
-
-    public required override convenience init() {
-        self.init(context: .init())
-    }
-
-    public init(context: Context) {
+    public override init() {
         super.init()
 
-        webView.configuration.processPool = context.processPool
```

Comment:
@kean would know more about the details. But I think the intention is reusing this `WKProcessPool` instance to speed up rendering. So, I don't think we should remove this line?

Reply:
`WKProcessPool` is deprecated now – Apple says it doesn't actually do anything